client_go18_test.go

```diff
@@ -15,12 +15,12 @@ import (
 var (
 	testTrace = &api.Trace{
 		Frames: []*api.Frame{
-			&api.Frame{
+			{
```

Comment:
Thanks for fix it. more simply :D

BTW, I should remove go
runtime version dependency on test.
Currently uses `errors.New` but that’s depends to line number per go versions. very ugly 😱